src/knot_protocol.c

```diff
@@ -98,7 +98,7 @@
 
 #define TYPE_ID_MASK(type)	(type & 0x000F)		
 
-static struct schema {
+static const struct schema {
```

Comment:
Please squash this commit into the initial commit (commit that introduced this change, I guess it belongs to this patchset)